pantheon-decoupled.php

```diff
@@ -44,3 +45,54 @@ function pantheon_decoupled_graphql_smart_object_cache() {
 }
 
 add_action('init', 'pantheon_decoupled_enable_deps');
+
+function decoupled_settings_init() { 
+
+	register_setting( 'wp-pantheon-decoupled', 'decoupled_settings' );
+
+	add_options_page( 'Pantheon Front-End Sites', 'Pantheon Front-End Sites', 'manage_options', 'wp-pantheon-decoupled', 'decoupled_options_page' );
```

Comment:
I think we could make the menu slug of `wp-pantheon-decoupled` more descriptive. Maybe something like 'pantheon-fes' or 'pantheon-front-end-sites'